src/rc.c

```diff
@@ -30,29 +33,29 @@ static void convertPWMtoRad()
   // Get Roll control command out of RC
   if (_rc_control.x.type == ANGLE)
   {
-    _rc_control.x.value = ((pwmRead(_params.values[PARAM_RC_X_CHANNEL]) - _params.values[PARAM_RC_X_CENTER])
-                           *2*_params.values[PARAM_RC_MAX_ROLL_MRAD])/_params.values[PARAM_RC_X_RANGE];
+    _rc_control.x.value = (float)((pwmRead(_params.values[PARAM_RC_X_CHANNEL]) - _params.values[PARAM_RC_X_CENTER])
```

Comment:
Again, just thinking that the RC module would only be responsible for returning normalized values, then the muxer would decide how to interpret them (apply the max roll/rollrate value based on the mode, etc.). Again, we can do that in a separate pull request